File: Backend/CVManagement.Domain/Entities/ExperienceInformation.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace CVManagement.Domain
{
    public class ExperienceInformation : Entity<int>
    {
        public string CompanyName { get; set; }
        public string City { get; set; }
        public string CompanyField { get; set; }
        public int CVId { get; set; }

        public void Delete()
        {
            IsDeleted = true;
        }
    }
}

File: Fronend/src/app/model/experience-information/ExperienceInformation.ts
export class ExperienceInformationModel
{
  id: number = 0;
  companyName: string = '';
  city: string = '';
  companyField: string = '';
  isDeleted: boolean = false;

  constructor(_companyName: string = '', _city:string = '', _companyField: string = ''){
    this.companyName = _companyName;
    this.city = _city;
    this.companyField = _companyField;
  }
}
